walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,143 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.Iterator;
 
 public final class FindMeetingQuery {
+
+  /* runs in O(n*m+p), where n is the number of events, m is the maximum number of attendees for any given event, and p is
```

Comment:
Add comments explaining what this function is supposed to do (Brief explanation of what it's trying to solve).